include/libdnf5/repo/repo.hpp

```diff
@@ -385,6 +372,21 @@ class Repo {
     std::unique_ptr<SolvRepo> solv_repo;
 
     WeakPtrGuard<Repo, false> data_guard;
+
+    bool loaded{false};
```

Comment:
Please move this member together with other privates, f.e. right below `bool expired{false};`